bats/fb-content-katello.bats

```diff
@@ -121,6 +121,12 @@ EOF
   tPackageExists subscription-manager || tPackageInstall subscription-manager
 }
 
+@test "disable puppet agent to prevent checkin from registering host to another org" {
+  if systemctl is-active puppet; then
+    run systemctl stop puppet
```

Comment:
This ignores the output. Is that what we want?

Reply:
i thought 'run' would capture the output if it failed, i'm not sure i get what you mean?

Reply:
https://github.com/sstephenson/bats#run-test-other-commands

Run allows you to use test a programs output, regardless of its exit code. It'll always exit with code 0 and stores the result in `$status` and the output in in `$output`. Since the code is 0, this test will silently fail. If you drop the `run`, then it'll automatically capture the output on failure in the test results.